plugins/kubernetes/app/models/kubernetes/release.rb

```diff
@@ -87,6 +87,20 @@ def update_status(release_doc)
       save!
     end
 
+    def pods
```

Comment:
Small detail: I'd suggest calling this `#fetch_pods`.

If you call it `#pods`, I'd assume that it's like an ActiveRecord association, and the results are cached.  So I could call `release.pods` several times without doing a round-trip to Kubernetes each time.  If you call it `fetch_pods`, that hints to me that calling it does a network roundtrip.